research/index.html

```diff
@@ -24,40 +24,47 @@
         <nav>
           <ul class="nav nav-pills float-right">
             <li class="nav-item">
-              <img src="image/quadcore_logo.png" class="nav-link active" href="#"><span class="sr-only"> ()
```

Comment:
Only one `nav-link` item should active. Two or more active items are not allowed.